components/automate-ui/src/app/pages/+compliance/+reporting/reporting.component.html

```diff
@@ -30,6 +30,20 @@
           </chef-click-outside>
         </chef-dropdown>
       </div>
+      <div class="copy-dropdown">
+        <chef-button class="dropdown-toggle" secondary (click)="toggleShareDropdown()">
+          <chef-icon>share</chef-icon>
+        </chef-button>
+        <chef-dropdown class="dropdown" [visible]="shareOptsVisible">
+          <chef-click-outside (clickOutside)="hideShareDropdown()" omit="dropdown-toggle">
+            <p>Copy this private URL to share:</p>
+            <div class="copy-input">
+              <input type="text" [value]="shareUrl" readonly />
+              <chef-clipboard [value]="shareUrl"></chef-clipboard>
+            </div>
+          </chef-click-outside>
+        </chef-dropdown>
+      </div>
```

Comment:
This adds the copy URL button next to the search bar. 
![Screen Shot 2019-09-09 at 11 15 33 AM](https://user-images.githubusercontent.com/1679247/64555904-35064500-d2f3-11e9-93dd-5424401487c5.png)